features/wasm-threads.yml

```
@@ -0,0 +1,7 @@
+name: WebAssembly threads and atomics
```

```suggestion
name: Threads and atomics
```

Alternatively, if we truly need to disambiguate:

```suggestion
name: Threads and atomics (WebAssembly)
```